portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,54 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+    <header>
+        <div class="homeheader">
+            <h1>Bria Fassler's Portfolio</h1>
+        </div>
+    </header>
+    <nav>
+        <div class="navbar">
+            <a href="index.html">Home</a>
+            <a href="dogs.html">Dogs</a>
+            <a href="sports.html">Sports</a>
+        </div>
+    </nav>
+    <aside>
+        <div class="homeaside">
+            <h2>More Info About Me!</h2>
+                <a href="https://www.linkedin.com/in/briafassler">Linkedin</a><br><br>
+                <a href="https://github.com/briafassler9/step.git">Github</a>
+                <p>Email: brifass11@aol.com</p>
+                <p>Phone Number: (443) 680-3550</p>
+        </div>
+    </aside>
+    <section>
+        <h1>About Me</h1>
+            <p>
+            Hi! Here are the basics about me. My name is Bria Fassler and I'm from Baltimore, Maryland. I'm currently 18 years old but on June 22nd, I'll be 19 so I'd say it's close enough. 
+            I'm pretty short, 5'1" to be exact, but I'm still hoping for a last minute growth spurt. I live with my mom and dad, but have an older sister who moved out a couple of years ago. 
```

Comment:
nit: Remove whitespace at the end of this line.

I'm still waiting for a growth spurt too ):